kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaProtocolHandler.java

```diff
@@ -226,6 +229,108 @@ public boolean test(NamespaceBundle namespaceBundle) {
 
     }
 
+    public static class TxnNamespaceBundleListener implements NamespaceBundleOwnershipListener {
```

Comment:
Could we merge this with the group namespace bundle listener?  If we have two listeners, it will check all topics twice.

Reply:
This a temporary solution, it may optimize in others PR.